Kitodo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -124,6 +124,10 @@
     <filter>
         <filter-name>requestControlFilter</filter-name>
         <filter-class>de.sub.goobi.helper.servletfilter.RequestControlFilter</filter-class>
+        <init-param>
+            <param-name>excludePattern</param-name>
+            <param-value>.*\.(js|css)\.jsf</param-value>
```

Comment:
This pattern looks a bit strange. There should no file name ending of `.js.jsf` or `.css.jsf` if I understood this pattern correct.